// discord-badges/src/badges.js
module.exports = {
	BUGHUNTER_LEVEL_1: {
		name: 'Discord Bug Hunter',
		url: '<:stt_bug_huner_1:896337630399189002>',
	},
	BUGHUNTER_LEVEL_2: {
		name: 'Discord Bug Hunter',
		url: '<:stt_bug_huner_2:896337630680211456>',
	},
	DISCORD_EMPLOYEE: {
		name: 'Discord Staff',
		url: 'https://cdn.discordapp.com/emojis/843414316202983435.png?v=1',
	},
	DISCORD_NITRO: {
		name: 'Discord Nitro',
		url: '<:stt_nitro:901398398769430558>',
	},
	EARLY_SUPPORTER: {
		name: 'Early Supporter',
		url: '<:stt_early_supporter:901398398828154880>',
	},
	HOUSE_BALANCE: {
		name: 'HypeSquad Balance',
		url: '<:stt_balance:896337630512427038>',
	},
	HOUSE_BRAVERY: {
		name: 'HypeSquad Bravery',
		url: '<:stt_bravery:896337630600503306>',
	},
	HOUSE_BRILLIANCE: {
		name: 'HypeSquad Brilliance',
		url: '<:stt_brillance:896337630457905162>',
	},
	HYPESQUAD_EVENTS: {
		name: 'HypeSquad Events',
		url: '<:stt_premium:894863362251427881>',
	},
	EARLY_VERIFIED_BOT_DEVELOPER: {
		name: 'Early Verified Bot Developer',
		url: '<:stt_verified_dev:896337630499848263>',
	},
	PARTNERED_SERVER_OWNER: {
		name: 'Partnered Server Owner',
		url: '<:stt_partner:896337630625673236>',
	},
	DISCORD_CERTIFIED_MODERATOR: {
		name: 'Discord Certified Moderator',
		url: '<:stt_shield:894863362658283550>',
	}
};
